### DISCUSSION TITLE
CIRA TROUBLESHOOTING

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6175] [Q&A] [answered]
Hi, how would I troubleshoot CIRA and figure out what is the cause?  I have a remote NUC device on a different network that has AMT and TLS.  Everything looks ok but seems like CIRA is not connecting.  Any suggestions?  Thx! 
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/538e47f7-2f8d-44e7-a4ae-93c5477324a8)
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/4a4471b6-58c6-4a85-9e60-3161a390a836)

**Comment by si458** [comment]
Have u checked the amt port is open on ur meshcentral side?
What's the output on the console say when u run meshcentral?

**Reply by mtran-88** [reply, marked as answer]
Hi Simon, thank you for replying!  I just found the issue.  User error!  I've mistyped the port number in the firewall.   :O)
